### DISCUSSION TITLE
Use known vars as rewrite value

### DISCUSSION THREAD

**Discussion by gk-fschubert** [radsecproxy/radsecproxy#126] [Q&A] [answered]
Is there a way to use known variables/attributes in the rewrite block as value? 
As far as i understood you can "only" use static values. 

It would be really good to reserve the public IP address of the client which is trying to authenticate in the RADIUS requests as custom attribute.

**Comment by fmauchle** [comment, marked as answer]
No, this is not supported in radsecproxy. See also #89

**Reply by gk-fschubert** [reply]
Would this functionality be accepted as a PR?
Since the original client IP isn't a standard RADIUS attribute but comes from connection metadata, it can't reuse the existing `addAttribute` path — so it would need new directives. I was thinking along these lines:

```
addMetaAttribute <attr>:<keyword>
addMetaVendorAttribute <vendor>:<subattr>:<keyword>
supplementMetaAttribute <attr>:<keyword>
supplementMetaVendorAttribute <vendor>:<subattr>:<keyword>
```

with keywords `source_ip`, `source_port`, and `client_name` inserted as string
